main_window/packet_parser.py

```diff
@@ -50,6 +52,18 @@ class SubpacketIds(Enum):
 }
 DEVICE_TYPE_TO_ID = {y: x for (x, y) in ID_TO_DEVICE_TYPE.items()}
 
+# Matches enum EventId in FLARE: radio.h
```

Comment:
If possible, might want to use the enum "DataEntryValues" in the data_entry_ids.py